pagopa/io-app
# chore: [IOPAE-1250] Hide error toasts on refresh session

## Short description
This PR avoids showing error toasts when the FL is active and the session has expired (401 HTTP).

<details open><summary>Details</summary>
<p>

| before | after |
| - | - |
| <video src="https://github.com/pagopa/io-app/assets/29163287/932f53a0-1ace-487e-a1b1-c9b65080305b" /> | <video src="https://github.com/pagopa/io-app/assets/29163287/980a414c-7344-4c6a-8b9e-1d50c11d53ee" /> | 

</p>
</details> 

## List of changes proposed in this pull request
- added guard for http 401
- bump `@pagopa/io-app-design-system` to `1.38.3`

## How to test
Enable FL and navigate to services tab. Do a search and wait for the session to expire. After renewing the session, you should not see error toasts
